server/api/tags_apis.py

```diff
@@ -26,6 +28,58 @@ def get(self):
             current_app.logger.critical(error_msg)
             return {"error": error_msg}, 500
 
+    @token_auth.login_required
```

Comment:
You should add the @tm.pm_only() decorator here as well.  This decorator means you must be logged in and have the Project Manager role to use the API.  I'd suggest only Project Managers can delete tags.  Without the decorator any logged in user could delete a tag.

Reply:
ok will add this.